src/main/java/org/b612foundation/adam/datamodel/AdamObject.java

```diff
@@ -0,0 +1,48 @@
+package org.b612foundation.adam.datamodel;
+
+import java.util.Objects;
+
+public class AdamObject {
+
+  /** Unique id for an object, generated on creation. */
```

Comment:
Comment nit: "generated on creation" no longer applies. Also, as far as astrodynamics library is concerned, the UUID is optional.

Reply:
Comment fixed. True.